src/dev/storybook/aliases.ts

```diff
@@ -13,6 +13,7 @@ export const storybookAliases = {
   ci_composite: '.ci/.storybook',
   cloud: 'x-pack/plugins/cloud/.storybook',
   coloring: 'packages/kbn-coloring/.storybook',
+  content_management: 'packages/kbn-content-management/table_list/src/config',
```

Comment:
nit: we could consider creating `packages/content-management` as a root, `packages/content-management/table_list` as a package, and then create `packages/content-management/.storybook` to hold the Storybook config.  Then we could point the Storybook config to scan `packages/content-mangement/**/*.stories.tsx`.

Storybook configs don't have to exist in a package, as they're run-time and have no use bundled.

Reply:
Ok will try that 👍